local-dev.sh

```diff
@@ -653,53 +683,32 @@ setup_database() {
                         exit 1
                     fi
                 fi
-                ;;
-            2)
-                # Could not determine status, ask user what to do
-                log_warning "Could not determine database status."
-                echo "You can either:"
-                echo "  1. Try to migrate (safe, preserves data)"
-                echo "  2. Reset database (destructive, deletes all data)"
-                echo
-                if ask_user_confirmation "Do you want to try migrating first"; then
-                    if migrate_database; then
-                        log_success "Database migrated successfully"
-                    else
-                        log_warning "Migration failed."
-                        if ask_user_confirmation "Do you want to reset the database? This will delete all data"; then
-                            log_info "Resetting database..."
-                            pnpm run db-reset || error_exit "Failed to reset database"
-                        else
-                            log_error "Database setup cancelled by user"
-                            exit 1
-                        fi
-                    fi
+            else
+                if ask_user_confirmation "Do you want to reset the database? This will delete all data"; then
+                    log_info "Resetting database..."
+                    pnpm run db-reset || error_exit "Failed to reset database"
                 else
-                    if ask_user_confirmation "Do you want to reset the database? This will delete all data"; then
-                        log_info "Resetting database..."
-                        pnpm run db-reset || error_exit "Failed to reset database"
-                    else
-                        log_error "Database setup cancelled by user"
-                        exit 1
-                    fi
+                    log_error "Database setup cancelled by user"
+                    exit 1
                 fi
-                ;;
+            fi
+            ;;
         esac
     fi
-    
+
     cd - >/dev/null
     log_success "Database setup completed"
 }
 
 # Start development server
 start_dev_server() {
     log_info "Starting development server..."
-    
+
     # Use turbo if available, otherwise use pnpm
     if command_exists turbo; then
-        turbo dev --filter=@reefguide/web-api --filter=@reefguide/app
+        turbo dev
```

Comment:
do we still need these filters? when I run `local-dev.sh`, it tries to run the capacity-manager
> @reefguide/capacity-manager:dev

Reply:
fixed by remove `dev` from capacity-manager and example-worker package.json files